releasedir/provider.go

```diff
@@ -103,6 +103,8 @@ func (p Provider) newBlobstore(dirPath string) boshblob.DigestBlobstore {
 		blobstore = boshblob.NewLocalBlobstore(p.fs, p.uuidGen, options)
 	case "s3":
 		blobstore = NewS3Blobstore(p.fs, p.uuidGen, options)
```

Comment:
If you have to add a switch statement to every method, I would prefer it the way it is. Why is the switch statement necessary? I was hoping that since they get called the same way that is wouldn't be necessary.